hydradx/notebooks/Omnipool/LP_returns_analysis.ipynb

```diff
@@ -0,0 +1,392 @@
+      "text/plain": [
+       "<Figure size 1440x360 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "price_path = []\n",
+    "modifier_list = [-0.25, 0, 0.25, 0.5, 0.75]\n",
+    "colors = pu.color_gradient(length=5, color1=(255, 0, 0), color2 = (0, 255, 0))\n",
+    "initial_price_sum = sum([price_list[0][tkn] for tkn in price_list[0]])\n",
```

Comment:
It is not meaningful to sum prices in this way. Prices are in units USD/BTC, USD/ETH, USD/DOT, etc, and summing them is not correct. Whatever is the highest price (BTC) will dominate the sum, even if BTC makes up a tiny portion of the Omnipool. Then the aggregate graph will be something close to the BTC graph but won't reflect the overall dynamics of the Omnipool.

The quickest fix is to just do this graph with BTC price since that's approximately what it is currently. We could also do the other assets individually, and graph the result for each asset.

If you want some aggregate measure, you can do something like take % change of each asset and then do a weighted average. But I think it's better to just look at each asset individually.

Reply:
This is resolved, I'm now looking at TKN/LRNA vs USD/LRNA prices.